### PR TITLE
adding new aws-terraform-qe profile

### PR DESCRIPTION
**Pull request by amalykhi**
CI tools PR https://github.com/openshift/ci-tools/pull/3540

### CODE REVIEW

**Comment by amalykhi** [comment]
Please review @yunjiang29 @yasun1

File: core-services/prow/02_config/generate-boskos.py [hunk 1]
```diff
@@ -43,6 +43,13 @@
         'us-east-1': 25,
         'ap-northeast-1': 5,
     },
+    'aws-terraform-qe-quota-slice': {
+        'ap-northeast-1': 2,
+        'us-east-1': 2,
+        'us-east-2': 2,
+        'us-west-1': 2,
+        'us-west-2': 2,
```
Comment (yasun1) [inline core-services/prow/02_config/generate-boskos.py:51]: I assume `2` is small for usage. The number at least > 3 for one is automation, another one is debugging, the third one is for any accident.

File: core-services/prow/02_config/generate-boskos.py [hunk 2]
```diff
@@ -43,6 +43,14 @@
         'us-east-1': 25,
         'ap-northeast-1': 5,
     },
+    'aws-terraform-qe-quota-slice': {
+        'ap-northeast-1': 3,
+        'ap-southeast-1': 3,
+        'us-east-1': 3,
+        'us-east-2': 3,
+        'us-west-1': 3,
+        'us-west-2': 3,
```
Comment (yunjiang29) [inline core-services/prow/02_config/generate-boskos.py:52]: @amalykhi This number means the concurrent jobs number, I'd suggest starting at 3 or 5, we can increase this value if quota issue appears

**Comment by yunjiang29** [comment]
@danilo-gemoli can you approve this PR? thanks

File: core-services/prow/02_config/generate-boskos.py (hunk 2 above)
Comment (amalykhi) [reply core-services/prow/02_config/generate-boskos.py]: Done

File: core-services/prow/02_config/_boskos.yaml [hunk 3]
```diff
@@ -646,6 +646,14 @@ resources:
   - us-west-2--aws-sd-qe-quota-slice-2
   state: free
   type: aws-sd-qe-quota-slice
+- names:
+  - ap-northeast-1
```
Comment (amalykhi) [inline core-services/prow/02_config/_boskos.yaml:650]: @yunjiang29 I have question. add _ap-northeast-1--aws-terraform-qe-quota-slice-0_
But when I execute _make boskos-config_ - the string after "1" is removed. What am I doing wrong?

**Pull request merged**